packages/web-pixels-extension/src/schemas/pixel-events.ts

```diff
@@ -604,17 +605,20 @@ export const pixelEvents = {
           metadata: {
             description:
               'Indicates whether the customer has consented to be sent marketing material via email. This property is only available if the shop has [upgraded to Checkout Extensibility](https://help.shopify.com/en/manual/checkout-settings/checkout-extensibility/checkout-upgrade).',
+            exclude_from_ruby_schema: true,
```

Comment:
I know this property was recently added by @rlake80s. Wondering if this should be made public?

Reply:
@manasa-se @kencwng Do either of you have context on whether the new `exclude_from_ruby_schema` and `server_event_supported` properties that @rlake80s added should be made public?

Reply:
I'd like to ship this PR relatively quickly and removing these properties doesn't seem like a risk.

I'll go ahead and remove all the ruby typing properties and we can come back to this topic the next time someone publishes changes.